js-miniapp-sample/src/services/uuid/reducers.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type { UUIDAction } from "./actions";
+import { SET_UUID } from "./types";
+
+type UUIDState = {
+  +uuid: ?string,
+};
+
+const defaultState: UUIDState = {
+  uuid: undefined,
+};
+
+const UUIDReducer = (
+  state: UUIDState = defaultState,
+  action: UUIDAction = {}
+): UUIDState => {
+  if (action !== undefined) {
```

Comment:
Instead of using two nested if statements, we could simplify this to:

```suggestion
  if (action !== undefined && action.type === SET_UUID) {
     return { ...defaultState, uuid: action.payload };
  }
  return state
```